frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -398,26 +401,15 @@ export function DataField(props: TextFieldProps) {
     language = 'yaml';
   }
   return (
-    <>
-      <Box borderTop={0} border={1}>
-        <Box display="flex">
-          <Box width="10%" borderTop={1} height={'1px'}></Box>
-          <Box pb={1} mt={-1} px={0.5}>
-            <InputLabel>{label}</InputLabel>
-          </Box>
-          <Box width="100%" borderTop={1} height={'1px'}></Box>
-        </Box>
-        <Box mt={1} px={1} pb={1}>
-          <Editor
-            value={value as string}
-            language={language}
-            onMount={handleEditorDidMount}
-            options={{ readOnly: true, lineNumbers: 'off' }}
-            theme={themeName === 'dark' ? 'vs-dark' : 'light'}
-          />
-        </Box>
-      </Box>
-    </>
```

Comment:
Assuming this function is accessible from plugins, we shouldn't change it to just ignore the `label` property here, otherwise we may break external plugins' UIs.